pkg/descheduler/evictions/evictions.go

```diff
@@ -124,7 +132,9 @@ func (pe *PodEvictor) EvictPod(ctx context.Context, pod *v1.Pod, opts EvictOptio
 			if pe.metricsEnabled {
 				metrics.PodsEvicted.With(map[string]string{"result": "maximum number of pods per node reached", "strategy": strategy, "namespace": pod.Namespace, "node": pod.Spec.NodeName}).Inc()
 			}
-			klog.ErrorS(fmt.Errorf("Maximum number of evicted pods per node reached"), "limit", *pe.maxPodsToEvictPerNode, "node", pod.Spec.NodeName)
+			err = fmt.Errorf("Maximum number of evicted pods per node reached")
```

Comment:
looks like you're declaring this `err` variable just to call `err.Error()` below. you can just pass the string value directly (and then you can drop the `var err error` declartion on L125)